### PR TITLE
fix(evals): update prompt template for summarization eval and rails mapping to use lower case "good" / "bad"

### PR DESCRIPTION
**Pull request by arizedatngo**
bug due to llm returning lowercase "good" / "bad" was causing issues with NaN value. Changed the mapping and prompt to adjust

### CODE REVIEW

File: src/phoenix/experimental/evals/templates/default_templates.py [hunk 1]
```diff
@@ -97,15 +97,15 @@
     [END DATA]
     Compare the Summary above to the Original Document and determine if the Summary is
     comprehensive, concise, coherent, and independent relative to the Original Document.
-    Your response must be a string, either Good or Bad, and should not contain any text
-    or characters aside from that. Bad means that the Summary is not comprehensive, concise,
-    coherent, and independent relative to the Original Document. Good means the Summary
+    Your response must be a string, either good or bad, and should not contain any text
+    or characters aside from that. The string bad means that the Summary is not comprehensive, concise,
+    coherent, and independent relative to the Original Document. The string good means the Summary
```
Comment (RogerHYang) [inline src/phoenix/experimental/evals/templates/default_templates.py:102]: maybe add some quotes to make it stand out more (at least for human eyes)
```suggestion
    Your response must be a string, either "good" or "bad", and should not contain any text
    or characters aside from that. The string "bad" means that the Summary is not comprehensive, concise,
    coherent, and independent relative to the Original Document. The string "good" means the Summary
```

**Review by RogerHYang — APPROVED** [review]

**Review by axiomofjoy — APPROVED** [review]

File: src/phoenix/experimental/evals/templates/default_templates.py (hunk 1 above)
Comment (arizedatngo) [reply src/phoenix/experimental/evals/templates/default_templates.py:102]: yeah i had that originally, but i'm afraid, the LLM might include them in the output. Happy to do either. That would also cause the bug with that notebook. What do you think @axiomofjoy

Comment (RogerHYang) [reply src/phoenix/experimental/evals/templates/default_templates.py:102]: good point. fortunately quotes in the response are stripped in or processing

Comment (arizedatngo) [reply src/phoenix/experimental/evals/templates/default_templates.py:102]: ahh okay, even better. Still learning the library

Comment (arizedatngo) [reply src/phoenix/experimental/evals/templates/default_templates.py:102]: Thanks for the commit!

**Comment by mikeldking** [comment]
Hey @arizedatngo thanks for the fix! Heads up the PR title goes into our release notes. Mind making it a bit clearer? You can do things like fix(evals) to group it too. Thanks again!

**Comment by arizedatngo** [comment]
Yeah can do!

**Pull request merged**